packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -56,6 +56,26 @@ export default class PciStoragesContainersAddController {
       archive: this.archive,
     });
     this.container.region = null;
+
+    this.preselectStepItem();
+  }
+
+  /**
+   * Use this to preselect an item regarding current step
+   * To preselect others item you have to complete the switch
+   */
+  preselectStepItem() {
+    const { steps } = this.redirectTarget;
+    const currentStep = (this.currrentStep || 0) + 1; // to have humane start count
+
+    if (steps) {
+      switch (currentStep) {
+        case 1:
+          this.container.offer = steps[`STEP_${currentStep}`];
+          break;
+        default:
```

Comment:
Why having a `switch` for only one specific case? (Also, `default` case does nothing)

Reply:
I let a function comment to explain why.
currently the preselect is only implemented for step1 but it will be completed for others steps in the future.

Others have only to implement the new steps (switch cases).

True, i don't care about.

Reply:
Here we can use `if/else` or `switch` statement.
But it's more relevant to use `switch` statement because we check cases with only one expression (`currentStep`).
It's also scalable, for others steps we have only to use the current implementation and add others preselect feature.

Reply:
It could be more relevant if we have already at least three different cases. Where a simple if statement could be enough. Aside of this, it brings code smell, regarding Sonarcloud report: https://sonarcloud.io/project/issues?id=ovh_manager&pullRequest=7219&resolved=false&types=CODE_SMELL

Reply:
I agree with @JeremyDec  here. If you are just waiting for content, it's okay to let the switch. If you don't and are just anticipating potential futur changes, it's just over-engineering and it makes the code harder to read. 

I think an if is enough, and if an evolution is needed, it should be done when it will be. If we are not 100% sure that we will have more steps, it's just code smell that we will have to deal with.